connexion/decorators/security.py

```diff
@@ -54,7 +53,7 @@ def wrapper(*args, **kwargs):
         logger.debug("%s Oauth verification...", request.url)
         authorization = request.headers.get('Authorization')  # type: str
         if not authorization:
-            logger.error("... No auth provided. Aborting with 401.")
+            logger.info("... No auth provided. Aborting with 401.")
```

Comment:
I decided to go for info here because it's the same severity used on the successful case.